### DISCUSSION TITLE
Disable Jetty Metrics

### DISCUSSION THREAD

**Discussion by sidhant92** [dropwizard/dropwizard#5633] [Q&A]
Is there a way by which we can disable the default jetty timers being pushed to /metrics.

Example: `io.dropwizard.jetty.MutableServletContextHandler.get-requests`

**Comment by joschi** [comment]
> Is there a way by which we can disable the default jetty timers being pushed to /metrics.

@sidhant92 That's not supported in Dropwizard out of the box.

You could implement a custom [`ServerFactory`](https://github.com/dropwizard/dropwizard/blob/v2.1.1/dropwizard-core/src/main/java/io/dropwizard/server/ServerFactory.java), see [`AbstractServerFactory`](https://github.com/dropwizard/dropwizard/blob/v2.1.1/dropwizard-core/src/main/java/io/dropwizard/server/AbstractServerFactory.java#L604-L607), which doesn't instrument the Jetty [`ServletContextHandler`].

If it's about a custom metrics reporter, you can probably add a custom [`MetricFilter`](https://www.javadoc.io/doc/io.dropwizard.metrics/metrics-core/4.2.0/com/codahale/metrics/MetricFilter.html) to it.

**Comment by sidhant92** [comment]
Does this make sense to keep it behind a flag?
